sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,2275 +1,2278 @@
+                "data-cy": "nocCode"
+              },
+              "type": "textfield",
+              "input": true,
+              "lockKey": true,
+              "autofocus": false,
+              "inputType": "text",
+              "inputMask": "",
+              "placeholder": "",
+              "prefix": "",
+              "suffix": "",
+              "multiple": false,
+              "defaultValue": "",
+              "protected": false,
+              "unique": false,
+              "persistent": true,
+              "hidden": false,
+              "clearOnHide": true,
+              "spellcheck": true,
+              "conditional": {
+                "show": null,
+                "when": null,
+                "eq": ""
+              }
+            },
+            {
+              "label": "SABC program code",
+              "description": "Format (XXX#) Mandatory field if using the 'offerings upload' feature. Otherwise optional.",
+              "tableView": true,
+              "validate": {
+                "pattern": "[[A-Z]{3}[0-9]{1}",
+                "customMessage": "Incorrect Format"
+              },
+              "key": "sabcCode",
+              "logic": [
+                {
+                  "name": "Disable SABC Input",
+                  "trigger": {
+                    "type": "javascript",
+                    "javascript": "result = data.hasOfferings;"
+                  },
+                  "actions": [
+                    {
+                      "name": "Disable Input",
+                      "type": "property",
+                      "property": {
+                        "label": "Disabled",
+                        "value": "disabled",
+                        "type": "boolean"
+                      },
+                      "state": true
+                    }
+                  ]
+                }
+              ],
+              "attributes": {
+                "data-cy": "sabcCode"
+              },
+              "type": "textfield",
+              "input": true,
+              "autofocus": false,
+              "inputType": "text",
+              "inputMask": "",
+              "placeholder": "",
+              "prefix": "",
+              "suffix": "",
+              "multiple": false,
+              "defaultValue": "",
+              "protected": false,
+              "unique": false,
+              "persistent": true,
+              "hidden": false,
+              "clearOnHide": true,
+              "spellcheck": true,
+              "conditional": {
+                "show": "",
+                "when": null,
+                "eq": ""
+              },
+              "labelPosition": "top",
+              "inputFormat": "plain",
+              "tags": [],
+              "properties": {},
+              "tooltip": "Please see our <a href='https://studentaidbc.ca/sites/all/files/school-officials/how-to-assign-program-codes.pdf'>job aid</a> to choose the appropriate code for your program."
```

Comment:
Here is a sample of a hyperlink that seems to have all the necessary attributes.
`noopener noreferrer` were a security recommendation and the style was created to be applied to all form.io links but seem to be applied even to some vue links.

```html
<a href="https://studentaidbc.ca/sites/all/files/school-officials/policy_manual.pdf" target="_blank" class="formio-href" rel="noopener noreferrer">StudentAid BC Policy Manual</a>
```